models/comments.model.js

```diff
@@ -28,12 +33,25 @@ exports.insertComment = (article_id, username, body) => {
                     msg: "404: User Not Found",
                 });
             }
-            return db.query(
-                "INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;",
-                [article_id, username, body]
-            );
-        })
-        .then(({ rows }) => {
-            return rows[0];
+            return db
+                .query("SELECT * FROM articles WHERE article_id = $1;", [
+                    article_id,
+                ])
+                .then(({ rows }) => {
```

Comment:
This is nested .then(), that's probably why Prettier created a pyramid of doom for you. Chain it after the first .then would fix it